### PR TITLE
add docs for gitpod integration

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

File: content/en/user-guide/integrations/gitpod/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      AWS_DEFAULT_REGION: "us-east-1"
+    init: |
+      cd /workspace
+      curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+      unzip awscliv2.zip
+      sudo ./aws/install
+      cd $THEIA_WORKSPACE_ROOT
+      python -m pip install awscli-local
+ports:
+  - port: 4526-4559
```
Comment (whummer) [inline content/en/user-guide/integrations/gitpod/index.md:47]:
```suggestion
  - port: 4510-4559
    onOpen: ignore
  - port: 4566
```

File: content/en/user-guide/integrations/gitpod/index.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    env:
+      AWS_ACCESS_KEY_ID: "test"
+      AWS_SECRET_ACCESS_KEY: "test"
+      AWS_DEFAULT_REGION: "us-east-1"
+    init: |
+      cd /workspace
+      curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+      unzip awscliv2.zip
+      sudo ./aws/install
+      cd $THEIA_WORKSPACE_ROOT
```
Comment (whummer) [inline content/en/user-guide/integrations/gitpod/index.md:44]: Curious about the `$THEIA_WORKSPACE_ROOT` env variable - wasn't able to find much information, is this documented somewhere on Gitpod side?

File: content/en/user-guide/integrations/gitpod/index.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    addCheck: true
+    # add a "Review in Gitpod" button as a comment to pull requests
+    addComment: false
+    # add a "Review in Gitpod" button to the pull request's description
+    addBadge: true
+```
+
+After adding the configuration, you can start your development & testing by creating [your workspace in GitPod](https://www.gitpod.io/docs/getting-started/#start-your-first-workspace). Upon creation, you will be able to see the LocalStack container running in the background (you can use `localstack status` to check the status of the container).
+
+For a simple demonstration, check out the [LocalStack GitPod demo](https://github.com/whummer/localstack-gitpod-demo) repository. Check out our [in-depth walkthrough over the demo](https://localstack.cloud/blog/2022-09-26-localstack-x-gitpod-run-cloud-applications-with-localstack-and-gitpod/) on our blog!
```
Comment (whummer) [inline content/en/user-guide/integrations/gitpod/index.md:78]:
```suggestion
For a simple demonstration, check out the [LocalStack GitPod demo](https://github.com/Gitpod-Samples/localstack-gitpod-demo) repository. Check out our [in-depth walkthrough over the demo](https://localstack.cloud/blog/2022-09-26-localstack-x-gitpod-run-cloud-applications-with-localstack-and-gitpod/) on our blog!
```

**Review by whummer — APPROVED** [review]
Nice extension to our integration docs @HarshCasper . 🎉 LGTM overall, just a few smaller nitpicks.. 👍 Thanks for adding this!

File: content/en/user-guide/integrations/gitpod/index.md (hunk 2 above)
Comment (HarshCasper) [reply content/en/user-guide/integrations/gitpod/index.md:44]: The reference I found for this is here: https://github.com/gitpod-io/gitpod/issues/9804#issuecomment-1121839026

Unfortunately, its not documented but is used to `cd` to the workspace root.

**Pull request merged**